### ISSUE TITLE
Cells are shifted on iOS 15+ after clicking on the last column header in `rtl` mode with set height.

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#9621]
### Description

After clicking on the header of the last column in the specific case cells are shifted.

**Configuration:**
```
height large enough that there's no vertical scroll bar,
layoutDirection: 'rtl',
iOS 15+
```
This case is related to our [RTL demo](https://handsontable.com/docs/layout-direction/#rtl-demo).
This bug does not exist on other devices and on the iPad. Tested on Samsung Galaxy A5 / Android 11, iPad 6g / iPadOS 15.1, iOS 14. 
Can be reproduced via BrowserStack with demo https://jsfiddle.net/rmvgjy7d/

### Steps to reproduce

1. Click on the last column header.
2. Sometimes you have to click twice as much on the left side of the header.
<img width="221" alt="Screenshot 2022-07-01 at 16 27 36" src="https://user-images.githubusercontent.com/30186818/176913887-fb2ca2c9-175d-478d-9aa3-81b60ed33fe0.png">

The bug does not occur when the configured height is so small that a vertical scroll appears. 
https://jsfiddle.net/pstaznhv/2/ height: 50 - ok

### Demo

https://jsfiddle.net/97pze80g/2/ layoutDirection: 'rtl' v12.1, height: 'auto' - broken
https://jsfiddle.net/97pze80g/3/ layoutDirection: 'rtl', v12.1 no height - ok

https://jsfiddle.net/cvno8qa3/ layoutDirection: 'rtl' v12.0 - ok
https://jsfiddle.net/6ej1tbyL/ layoutDirection: 'rtl', v12.0 no height - ok

### Your environment
* Handsontable version: 12.1
* Browser name and version: Safari 15.5, Chrome
* Operating system: iOS 15+ (latest)

**Comment by budnix** [comment]
I think that it's not a regression. I was able to reproduce the bug using the first version that was released with RTL support (v12.0).

**Comment by aninde** [comment]
This issue will be fixed by the upcoming v12.1.3 https://jsfiddle.net/nrq83g1e/. Tested on real devices and on BrowserStack: 2 versions of iOS and iPadOS 15, 14, and 13. Real devices: Samsung Galaxy A5 / Android 11, iPhone SE (2020) / iOS 15.6.1, iPad 6 / iPadOS 15.6.1

**Comment by AMBudnik** [comment]
Thank you for rechecking that, @aninde 

I'm closing this issue as solved in Handsontable v12.1.3
